templates/includes/sidebar.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<ul>
+    <li>
+        <a href="{% url 'taxi:index'%}">Home page</a>
+    </li>
+    <li>
+        <a href="#">Manufacturers</a>
+    </li>
+    <li>
+        <a href="#">Cars</a>
+    </li>
+    <li>
+        <a href="#">Drivers</a>
+    </li>

```

Comment:
The href attributes for the Manufacturers, Cars, and Drivers links are not yet implemented. If these pages are not yet ready, consider commenting these lines out or adding a placeholder URL until the corresponding functionality is ready.